ros_ws/toad.sh

```diff
@@ -118,7 +118,7 @@ case $1 in
                     exit 1
                 fi
                 export DISPLAY=$(getActiveDisplay)
-                ARGUMENTS=""
+                ARGUMENTS="world:=$LAUNCHTRACK"
                 if [[ "$3" =~ "nogui" ]]; then
                     ARGUMENTS="gui:=false "
```

Comment:
nogui würde hier die fertigen Arguments überschreiben, daher muss das ein mal in ARGUMENTS="$ARGUMENTS gui:=false" geändert werden

Reply:
gutes auge ;)